src/nanoemoji/reorder_glyphs.py

```diff
@@ -84,13 +93,25 @@ def apply(self, font: ttLib.TTFont, value: otBase.BaseTable):
                 _sort_by_gid(font.getGlyphID, coverage_entry.glyphs, None)
 
 
-# (Type, Optional Format) => List[ReorderCoverage]
+@dataclass(frozen=True)
```

Comment:
I'm curious why we need a different type? - I think my inclination would have been to add an optional callable that post-processes the parallel list entries to the original ReorderCoverage. I don't object to this approach, just curious.